src/machine/board_rpi3.go

```diff
@@ -0,0 +1,182 @@
+// +build rpi3
+
+package machine
+
+import (
+	"device/arm"
+)
+
+//specific to raspberry pi 3
+var MemoryMappedIO = uintptr(0x3F000000)
```

Comment:
This should be changed to be a constant to simplify things.

Reply:
Done.